app/javascript/packs/calendar.js

```diff
@@ -1,9 +1,123 @@
+import '../../../app/javascript/stylesheets/_variables.scss'
+
+const commonCalendarOptions = {
+  plugins: ['interaction', 'timeGrid', 'dayGrid'],
```

Comment:
This line breaks the initialization. 
If you remove/comment it the calendar renders just fine.
Is it nescesearry? If it is, we should check the documnetation on how to use plugins.

```suggestion
 //  plugins: ['interaction', 'timeGrid', 'dayGrid'],
```
![Screenshot from 2021-02-07 16-25-58](https://user-images.githubusercontent.com/48379676/107151111-306b1380-6961-11eb-90ee-84b4aa43811a.png)

Reply:
not at all, totally agree.